### ISSUE TITLE
Amount of effort to update to AppMetrics 1.3

### ISSUE THREAD

**Issue by edwardmarbry** [alhardy/AppMetrics.Owin#2]
Looking through the source, finding it difficult to piece together dependencies.  Realistically how much effort would it be for me to update this project to be compatible with AppMetrics 1.3?

**Comment by alhardy** [comment]
The source is mostly just owin middleware , what dependencies are you referring to exactly?

How much effort? I don't think too much, however I would suggest waiting for AppMetrics 2.0.0 which almost there.

**Comment by johnduhart** [comment]
@alhardy At work we may be looking at integrating AppMetrics to our application, but we'd want to integrate using OWIN.

Would you accept a PR upgrading the projects to use the new project format, as well as dependencies?

**Comment by edwardmarbry** [comment]
I am also looking to integrate using OWIN but still forced to use .net 4.6.2 on a few projects.

I pulled the repo down and proceeded to update the nuget dependancies to the most recent versions(App.Metrics 1.3.0 ) and updated build requirement to 4.6.2.  To see what would break.

First thing was serializers(IMetricDataSerializer etc.) are now in App.Metrics.Formatters.Json. So I added the latest version of that.

Where I am at now, I am getting some reference conflicts with IMetricsHostBuilder and StringReporter being obsolete.  "Replaced with formatting packages which can be used with the report runner".  I am currently searching through docs and source to figure out what that actually entails.

Dependencies being what classes use and are used by which classes and how they interact in general.

**Comment by alhardy** [comment]
@johnduhart  Yes definitely, but I'd suggest waiting for 2.0.0. At the moment I'm in the process of removing dependencies on Microsoft's logging and DI extensions. It's close but probably 2-3 weeks from pushing an alpha version out. If you're keen to start on it before you could start working off the Dev packages on myget.

@Shinjifei That's part of the reason why I'm suggesting to wait for 2.0.0, that package, as well as others, are now targetting net452 and netstandard1.6

**Comment by johnduhart** [comment]
@alhardy Glad to hear it, fewer dependencies will make AppMetrics more palatable. I'll keep watch on the main repo.
